src/seals/util.py

```diff
@@ -37,7 +41,9 @@ def __init__(self, env, discard_terminal_observation=True, reset_reward=0.0):
         self.reset_reward = reset_reward
         self.previous_done = False  # Whether the previous step returned done=True.
 
-    def step(self, action):
+    def step(
+        self, action: WrapperActType
+    ) -> tuple[ObsType, SupportsFloat, bool, bool, dict[str, Any]]:
```

Comment:
Are the type hints here meant to be `Tuple` and `Dict` (from `typing`) instead of `tuple`, `dict` ?

Reply:
From Python 3.9 onward something like `dict[str, Any]` is supported and recommended. Since we are still on Python 3.8 with this library we should use `Dict[str, Any]` instead.

More details here: https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc